README.md

```diff
@@ -1,4 +1,4 @@
-![Macrosynergy](docs/source/_static/MACROSYNERGY_Logo_Primary.png)
+![Macrosynergy](https://raw.githubusercontent.com/macrosynergy/macrosynergy/develop/docs/source/_static/MACROSYNERGY_Logo_Primary.png?raw=True)
```

Comment:
Should this be the `main` branch for the [URL](https://raw.githubusercontent.com/macrosynergy/macrosynergy/develop/docs/source/_static/MACROSYNERGY_Logo_Primary.png?raw=True) instead of the develop? Otherwise looks good.

Reply:
Yes, made the change